tests/test_utils.py

```diff
@@ -119,26 +128,28 @@ def test_url_handler(url_type, url):
         assert is_opendap_url(processed)
 
 
+@pytest.mark.parametrize(
+    ("netcdfs", "argc"),
+    [([local_path("tiny_daily_pr.nc"), local_path("tiny_daily_prsn.nc"),], 3,)],
+)
+def test_collect_args_local(wps_test_collect_args, netcdfs, argc):
+    params = ";".join([f"file={nc}" for nc in netcdfs]) + f";argc={argc};"
+    run_wps_process(wps_test_collect_args, params)
```

Comment:
The actual test for `local` and `online` versions of this test basically do the same thing after the params setup. We could consider saving a bit of code by having these two lines be a method.